test/runtests.jl

```diff
@@ -547,63 +546,71 @@ end
             # P. Schimitt, et. al
             # A comparison of six methods for missing data imputation
             iris = dataset("datasets", "iris")
-            iris2 = @from i in iris begin
-                    @where i.Species == "versicolor" || i.Species == "virginica"
-                    @select {i.SepalLength, i.SepalWidth, i.PetalLength, i.PetalWidth}
-                    @collect DataFrame
-            end
-            data = Array(iris2)
+            iris2 = filter(row -> row[:Species] == "versicolor" || row[:Species] == "virginica", iris)
+            data = Array(iris2[:, [:SepalLength, :SepalWidth, :PetalLength, :PetalWidth]])
             num_tests = 100
 
             @testset "Iris - 0.15" begin
-                X = add_missings(data', 0.15)
+                X = add_missings(data, 0.15)
 
                 knn_nrmsd, mean_nrmsd = 0.0, 0.0
 
                 for i = 1:num_tests
-                    knn_imputed = impute(copy(X), Impute.KNN(; num_nn=1, missing_neighbors_threshold=0.5,
+                    knn_imputed = impute(copy(X), Impute.KNN(; k=2, threshold=0.5,
```

Comment:
Don't forget that it'll be easier to read if you do:

```
imp = Impute.KNN(;
    k=2,
    threshold=0.5,
    dist=Euclidean(),
    context=Context(; limit=1.0),
)
impute(copy(X), imp)
```
or just
```
impute(copy(X), Impute.KNN(; k=2))
```
Since the rest are just the defaults anyways.
```